python-sdk/docs/astro/sql/operators/load_file.rst

```diff
@@ -221,6 +221,17 @@ Loading to MS SQL
 .. note::
    We do not support loading Unicode data to SQL Server due to limitations on the underlying ``pymssql`` library
 
+Loading to Duckdb
+~~~~~~~~~~~~~~~~~
+
+``load_file`` can load data to duckdb. If the database file is not specified in connection, it will be created
+(the file extension may be .db, .duckdb, or anything else). The special value :memory: (the default) can be used to
```

Comment:
What determines the file extension?

Reply:
It may be worth to highlight that the value `memory` should be set in the connection, so users don't expect to set it in the `load_file` task itself.

Reply:
<img width="599" alt="Screenshot 2023-02-02 at 10 03 44 PM" src="https://user-images.githubusercontent.com/8670962/216381035-3cf9694d-2f71-4e40-9d9e-e70f21832344.png">
In connections if the file is kept empty it will use `memory`.

Reply:
@tatiana these are the supported extensions: https://duckdb.org/docs/extensions/overview.html

Reply:
@sunank200 thanks, based on what you wrote - we may want to rephrase the paragraph - please, take a look at the suggestion :)

Reply:
Done https://github.com/astronomer/astro-sdk/pull/1695/commits/e09109bc9c1daa83b99b36608ed066ef95a2e3e4